testsuites/CBLTester/CBL_Functional_tests/APITests/test_scopesCollections.py

```diff
@@ -0,0 +1,90 @@
+import pytest
```

Comment:
Maybe it's worth adding a setup/teardown fixture so everything that's created is cleanup at the end of test. That way, in the future it'll be easier to remove the clusters reset at the beginning of every test.

Reply:
The database is deleted at teardown from conftest.py. We can create a custom fixture for individual tests.